assets/helpers/abTests/abtest.js

```diff
@@ -62,7 +83,7 @@ function getMvtId(): number {
 
   let mvtId = cookie.get(MVT_COOKIE);
 
-  if (!mvtId) {
+  if (mvtId === null || mvtId === undefined) {
```

Comment:
nice, if you want to be really thorough, you could also check it's between 0-999999 inclusive ;)

Reply:
I added more validations here: https://github.com/guardian/support-frontend/commit/2478bcf